### ISSUE TITLE
distinction between galaxy position angle and beta

### ISSUE THREAD

**Issue by jchiang87** [LSSTDESC/gcr-catalogs#611]
I wanted to note a possible inconsistency regarding the galaxy ellipticity components and galaxy position angles described in the [current schema](https://github.com/LSSTDESC/gcr-catalogs/blob/10b9cce2b564cb8d8c27c613fd8098373319b07c/GCRCatalogs/SCHEMA.md).

@rmjarvis describes in [slack](https://lsstc.slack.com/archives/CHXKSF3HC/p1670190337342839) a distinction between `beta` and galaxy position angle:
```
beta is the angle of the galaxy major axis CCW from the +x axis.  In sky coordinates this means it is the angle
North of West.  (West being +x direction and North from there being CCW.)
I believe PA is traditionally defined as the angle East of North.  So with these definitions, beta = PA + 90.
```
I think this means that the `ellipticity_1` computed [here](https://github.com/LSSTDESC/gcr-catalogs/blob/3fa5f6b92abf3af26aa30d4baa1ab06b0925b323/GCRCatalogs/cosmodc2.py#L87) should instead be
```
return ellipticity*np.cos(2.0*pos_angle + np.pi)
```
and similarly for `ellipticity_2`, etc..

**Comment by rmjarvis** [comment]
Except the schema you linked to defines `position_angle` as "Position angle (arctan(E2/E1)), for galaxy, lensed", which is not the same as the traditional astronomy definition of [position angle](https://en.wikipedia.org/wiki/Position_angle).  So this might not be a bug.

**Comment by jchiang87** [comment]
Right, which is why I wondered if we needed the 90 degree shift to begin with.  So if it's not a bug here, then we don't need that 90 degree shift in skyCatalogs and imSim?  Regardless, it was applied in the DC2 sims.

**Comment by jchiang87** [comment]
A related issue is that `arctan(E2/E1)` doesn't necessarily get the right quadrant for the angle, so it's not clear to me that it's a useful definition since it has the same value for `E1-> -E1, E2-> -E2`, which has the same effect as adding that 90 degrees.

**Comment by rmjarvis** [comment]
My point was just that I don't know if the `pos_angle` that is being applied here is the same one we have in imsim.  If it is the same PA, then I think both (1) the schema here has the wrong definition of position_angle and (2) the code to calculate e1,e2 from it is wrong.
